commands/files.rb

```diff
@@ -0,0 +1,42 @@
+require 'json'
+require_relative '../lib/const'
+require_relative '../lib/convert_size'
+
+class Command
+  def self.files(pkg)
+    # Check if the package is even installed first, as this is the most likely reason we cannot find a filelist.
+    device_json = JSON.load_file(File.join(CREW_CONFIG_PATH, 'device.json'))
+    if device_json['installed_packages'].none? { |entry| entry['name'] == pkg.name }
+      puts "Package #{pkg.name} is not installed.".lightred
+      return
+    end
+
+    # We can't do anything if we don't have the filelist.
+    unless File.file?(filelist_path = File.join(CREW_META_PATH, "#{pkg.name}.filelist"))
+      puts "Package #{pkg.name} does not have a filelist :(".lightred
+      return
+    end
+
+    # Print the name and description of the package.
+    puts pkg.name.lightgreen + ": #{pkg.description}".lightblue
```

Comment:
I would think you would want the `:` lightgreen instead of lightblue.

Reply:
No, this mirrors the current behaviour and is how print_current_package does it.

Reply:
Just sayin'...